packager/config/nginx.conf.live.conf.template

```diff
@@ -11,12 +11,19 @@ vod_dash_profiles urn:mpeg:dash:profile:isoff-live:2011;
 expires off;
 vod_upstream_location "";	
 
-location ~ \/(?<live_type>live|recording|live-stg)\/[^\/]+\/p\/(?<partnerId>\d+)\/e\/(?<entryId>[^\/]+(?<entryHash>\w))\/(sd\/(?<segmentDuration>\d+)\/)?t\/(?<token>[^\/]+) {
+map $user_type $overrideClipTo{
+	"user" "";
+	default '{"clipFrom":0, "clipTo":0}';
```

Comment:
shouldn't it be clipTo: MaxInt?

Reply:
After a short conversation with Eran he said he wants to change it so that if "clipTo" value is 0 it will just disregard.